### PR TITLE
Feature/885 relevant users events portals

### PR DESCRIPTION
**Pull request by Tguntenaar**
_Resolves #885_

Adds the changes made to devhub portal in PR [845](https://github.com/NEAR-DevHub/neardevhub-bos/pull/845) to the events-committee.

[video.webm](https://github.com/user-attachments/assets/9a2c9c76-7b2e-4659-8c41-a2a5d87098ac)

### CODE REVIEW

**Comment by Tguntenaar** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Tguntenaar Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Are you going to win race-of-sloths this month? If so, you should speed up! Run, Sloth, run!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Tguntenaar)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Tguntenaar)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: instances/events-committee.near/widget/devhub/components/molecule/SimpleMDE.jsx [hunk 1]
```diff
@@ -139,7 +138,13 @@ function getSuggestedAccounts(term) {
...
 
   term = (term || "").replace(/\W/g, "").toLowerCase();
-  const limit = 5;
+  let limit = 5;
+  if (term.length < 2) {
+   results = [${sortedRelevantUsers
+     .map((u) => "{accountId:'" + u + "', score: 60}")
+     .join(",")}];
+    limit = ${5 + sortedRelevantUsers.length};
+  }
```
Comment (Megha-Dev-19) [inline instances/events-committee.near/widget/devhub/components/molecule/SimpleMDE.jsx:147]: shouldn't we use reuse `SimpleMDE` from the devhub instance?

File: instances/events-committee.near/bos.config.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
   "account": "events-committee.near",
+  "accounts": {
+    "deploy": "events-committee.near",
+    "signer": "events-committee.near"
+  },
```
Comment (Megha-Dev-19) [inline instances/events-committee.near/bos.config.json:6]: Why do we need this?

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 2

**Comment by race-of-sloths** [comment]
### 🌟 Score recorded!

@Megha-Dev-19, thank you for scoring this pull request in the Race of Sloths!

File: instances/events-committee.near/widget/devhub/components/molecule/SimpleMDE.jsx (hunk 1 above)
Comment (Tguntenaar) [reply instances/events-committee.near/widget/devhub/components/molecule/SimpleMDE.jsx:147]: Yes defenitly, we can do it in the next iteration of improvements.

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 2!
You have received 20 Sloth points for this contribution

**Pull request merged**